layouts/partials/theme-toggle.html

```diff
@@ -1,3 +1,10 @@
+{{- $navbar := .navbar | default false -}}
+{{- $height_px := 12 -}}
+{{- $height_tw := "hx-text-xs" -}}
+{{- if $navbar -}}
+  {{- $height_px = 24 -}}
+  {{- $height_tw = "hx-text-sm" -}}
+{{- end -}}
```

Comment:
just curious where are `12` and `24` coming from?

Reply:
Great question!  I've been offline for a few weeks, and honestly, I'd have to say it is likely a best estimate based on testing that made things look "right."

If those values can be computed from other inputs in the theme configuration, I'm certainly willing to explore that.  Do you have any guidance on how we might calculate a sane value for this?